api/cloud-control/v1beta1/vpcpeering_types.go

```diff
@@ -37,6 +36,17 @@ const (
 	VirtualNetworkPeeringStateDeleting     = "Deleting"
 )
 
+const (
+	VpcPeeringConnectionStateReasonCodeInitiatingRequest = "Initiating-Request"
+	VpcPeeringConnectionStateReasonCodePendingAcceptance = "Pending-Acceptance"
+	VpcPeeringConnectionStateReasonCodeActive            = "Active"
+	VpcPeeringConnectionStateReasonCodeDeleted           = "Deleted"
+	VpcPeeringConnectionStateReasonCodeRejected          = "Rejected"
+	VpcPeeringConnectionStateReasonCodeFailed            = "Failed"
+	VpcPeeringConnectionStateReasonCodeExpired           = "Expired"
+	VpcPeeringConnectionStateReasonCodeProvisioning      = "Provisioning"
+	VpcPeeringConnectionStateReasonCodeDeleting          = "Deleting"
+)
```

Comment:
Please correct me if I am wrong, but these statuses will only be used for the AWS peering, correct?
If that is the case, perhaps they should be named in a way that reflects that, just to remove potential future confusion.